SeleniumTestXUnit/Tests/Features/Project/Update.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Project Update
+    As a logged in user, the user should be able to update a project.
+    
+    Background:
+        Given the user is logged in
+
+    @Regression @go.to.login.page @create.project @delete.project
+    Scenario: Edit a project
+        When the user hovers over a Project
+            And clicks on the menu list 
+            And clicks on the edit option
```

Comment:
These 3 steps should be handled on a single step that hovers over the project, expands the menu and selects an option

Reply:
Done